src/components/calculator/PointsDisplay.tsx

```diff
@@ -29,49 +35,296 @@ function displayPercent(points: number): string {
 
 export function ExplanationCard(props: {
   points: number
+  lowerBound: number
+  upperBound: number
   repeatedEvent: boolean
   riskBudget: number
   riskBudgetSetter: (newValue: number) => void
+  data: CalculatorData
 }): React.ReactElement {
   const points = props.points
 
-  const activeRiskLevel = howRisky(points, props.riskBudget)
+  const [displayCalculationExplanation, setCalculationExplanation] = useState(
+    false,
+  )
+
+  const locationRisk = calculateLocationPersonAverage(props.data) || 0
+  const personRiskEach = Math.round(
```

Comment:
Why round here? Rounding should be handled in the display function (with `displayPoints`)

Reply:
Good call. I was borrowing code from elsewhere that was probably display code.

Reply:
![image](https://user-images.githubusercontent.com/32290/96958522-8ef3ab00-14b2-11eb-9c0f-c98d803bac7f.png)

Reply:
`displayPoints` rounds to 1 sig fig. Here we aren't rounding it to sig figs yet. So we need to round it a different way. 

So I think we keep it as-is.